alf/metrics/metrics_test.py

```diff
@@ -123,6 +123,79 @@ def testMetric(self, metric_class, num_trajectories, expected_result,
             self.assertEqual([0.0] * 2 if vector_reward else 0.0,
                              metric.result())
 
+    def test_average_per_step(self):
+        metric = AverageEnvInfoMetric(
```

Comment:
Suggest defining a new example metric class in this test case where `_extract_metric_values` return other custom dict or namedtuple other than `env_info`.

Reply:
Good point. Updated the unit tests with a cusotimzed metric class with customized value extraction.